hive-llap/llap.sh

```diff
@@ -0,0 +1,353 @@
+        --clobber
+    bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.txn.manager' --value 'org.apache.hadoop.hive.ql.lockmgr.DbTxnManager' \
+        --clobber
+    bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.support.concurrency' --value 'true' \
+        --clobber
+    bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.llap.io.allocator.alloc.min' --value '256Kb' \
+        --clobber
+    bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.compactor.initiator.on' --value 'true' \
+        --clobber
+    bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.compactor.worker.threads' --value '1' \
+        --clobber
+    bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.tez.container.size' --value "${EXECUTOR_SIZE}" \
+        --clobber
+
+    if [[ -z "$ADDITIONAL_MASTER" ]]; then
+            bdconfig set_property \
+            --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+            --name 'hive.zookeeper.quorum' --value "${LLAP_MASTER_FQDN}:2181" \
+            --clobber
+        bdconfig set_property \
+            --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+            --name 'hive.zookeeper.client.port' --value '2181' \
+            --clobber
+    fi
+}
+
+# add configurations to core-site for LLAP; add zookeeper details
+function configure_core_site(){
+    echo "configure core-site.xml..."
+
+    if [[ -n "$ADDITIONAL_MASTER" ]]; then
+        bdconfig set_property \
+            --configuration_file "${HADOOP_CONF_DIR}/core-site.xml" \
+            --name 'hadoop.registry.zk.quorum' --value "\${hadoop.zk.address}" \
+            --clobber
+        bdconfig set_property \
+            --configuration_file "${HADOOP_CONF_DIR}/core-site.xml" \
+            --name 'hadoop.registry.zk.root' --value "/registry" \
+            --clobber
+        bdconfig set_property \
+            --configuration_file "${HADOOP_CONF_DIR}/core-site.xml" \
+            --name 'hadoop.registry.rm.enabled' --value "true" \
+            --clobber
+    else
+        bdconfig set_property \
+            --configuration_file "${HADOOP_CONF_DIR}/core-site.xml" \
+            --name 'hadoop.registry.zk.quorum' --value "${LLAP_MASTER_FQDN}:2181" \
+            --clobber
+        bdconfig set_property \
+            --configuration_file "${HADOOP_CONF_DIR}/core-site.xml" \
+            --name 'hadoop.registry.zk.root' --value "/registry" \
+            --clobber
+        bdconfig set_property \
+            --configuration_file "${HADOOP_CONF_DIR}/core-site.xml" \
+            --name 'hadoop.registry.rm.enabled' --value "true" \
+            --clobber
```

Comment:
looks like the same config is set in both if and else blocks. Can be moved outside.

Reply:
fixed